Resources/Prototypes/Entities/Structures/Specific/anomalies.yml

```diff
@@ -179,3 +179,38 @@
     anomalyContactDamage:
       types:
         Radiation: 10
+
+
+- type: entity
+  id: AnomalyIce
+  parent: BaseAnomaly
+  suffix: Ice
+  components:
+  - type: Sprite
+    sprite: Structures/Specific/Anomalies/ice_anom.rsi
+    layers:
+    - state: anom
+      map: ["enum.AnomalyVisualLayers.Base"]
+    - state: pulse
+      map: ["enum.AnomalyVisualLayers.Animated"]
+      visible: false
+  - type: PointLight
+    radius: 2.0
+    energy: 2.5
+    color: "#befaff"
+    castShadows: false
+  - type: Anomaly
+    anomalyContactDamage:
+      types:
+        Cold: 10
+  - type: IceAnomaly
+    supercriticalExplosion: Cryo
+    explosionTotalIntentsity: 600
+    explosionDropoff: 5
+    explosionMaxTileIntensity: 100
```

Comment:
consider messing around with the `explosionui` command to get something a bit more dramatic. Explosion is cool, but maybe have it weaker with a bigger radius? Just some food for thought; ignore if you like it.